lib/skylight/user_config.rb

```diff
@@ -14,7 +14,18 @@ def initialize
     end
 
     def file_path
-      File.expand_path(ENV["SKYLIGHT_USER_CONFIG_PATH"] || "~/.skylight")
+      config_path = ENV["SKYLIGHT_USER_CONFIG_PATH"] ||
+        begin
+          require "etc"
+          home_dir = File.expand_path("~") || Etc.getpwuid.dir || (ENV["USER"] && File.expand_path("~#{ENV['USER']}"))
+          if home_dir
+            File.join(home_dir, ".skylight")
+          else
+            ""
+          end
+        end
+      # Must return something that is compatible with `File.exist?`
+      config_path.empty? ? config_path || File.expand_path(config_path)
```

Comment:
Syntax error, should be `:` not `||`.

I'd be ok with raising here if we don't have a path. We can then tell people to set `SKYLIGHT_USER_CONFIG_PATH`.